blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/IOContainer.java

```diff
@@ -85,18 +86,20 @@ static synchronized void resetInstance() {
      * @param driver
      * @param isRecordingDelayedMetrics
      */
-    private IOContainer(DriverType driver, boolean isRecordingDelayedMetrics) {
+    private IOContainer(DriverType driver,
+                        boolean isRecordingDelayedMetrics,
+                        boolean isDtxIngestBatchEnabled) {
```

Comment:
This change actually bothers me more than the change from LocatorIO to DLocatorIO in DBasicMetricsRWIntegrationTest. But I can't think of a better way (but still simple) way to do it.